docs/terminal/williamtown.md

```diff
@@ -92,4 +92,17 @@ Voiceless coordination is in place from WLM TCU to WLM ADC for arrivals cleared
 
 !!! phraseology
     <span class="hotline">**WLM TCU** -> **WLM ADC**</span>: "via WLM, ZULU, Close formation of 5, do you have any restrictions or requirements?”  
-    <span class="hotline">**WLM ADC** -> **WLM TCU**</span>: "ZULU, no restrictions or requirements, A040"  
\ No newline at end of file
+    <span class="hotline">**WLM ADC** -> **WLM TCU**</span>: "ZULU, no restrictions or requirements, A040"  
+
+
+## Miscellaneous
+### Stereo Approaches (Fast Jet Aircraft)
+Fast Jet Aircraft typically arrive back into Willy from Gate 6, at 'F125', as such you can assign them the stereo approach to arrive back into Willy.
+
+!!! Example
```

Comment:
I think I made some suggestions for this whole section in the first review that haven't come across yet

Reply:
Yeah, when I do the commit suggestion it appears it doesnt change it will go back and fix it,